src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -238,6 +240,20 @@ public function setMethods($methods)
     }
 
     /**
+     * Adds a resource for this collection.
+     *
+     * @param ResourceInterface $resource
+     *
+     * @return $this
+     */
+    private function addResource(ResourceInterface $resource)
```

Comment:
Actually, shouldn't this method be public? Otherwise I fail to see how `$resources` should ever be changed.